submissions/rubyAdeniregun/HTML/productpage.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>DUNA STORES</title>
+</head>
+<body>
+   <h1><bstrong>DUNA STORES</strong></h1> 
+   <nav>
+       <ul>
+           <li><a href="#">Home</a></li>
+           <li><a href="#">Back</a></li>
+           <li><a href="#">Categories</a></li>
+       </ul>
+   </nav>
+   <hr>
+   <section>
+    <h2>Fresh Body Wash (Organic and Skin Brightening Black Soap)</h2>
+    <h3><em>Brand: Skin Perfection Organics</em></h3>
+   </section>
+   <!--This section is for product details-->
+   <section>
+       <h3>Product Details</h3>
+       <img src="C:\Users\CRM ABUJA\Desktop\Dufuna-Fem\submissions\Images\IMG-20201001-WA0129.jpg" alt="Fresh Body Wash" style="width: 200px; height: 150px;">
```

Comment:
Since your images are now in your submissions folder, you can simply do   "../images/img-name" as the img src.